pom.xml

```diff
@@ -67,10 +67,43 @@
 			<artifactId>spring-restdocs-mockmvc</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
```

Comment:
what are all these dependencies needed for?

Reply:
It is well described here: https://stackoverflow.com/questions/43574426/how-to-resolve-java-lang-noclassdeffounderror-javax-xml-bind-jaxbexception-in-j

Reply:
From Spring